pkg/operator/const.go

```diff
@@ -9,4 +9,8 @@ const (
 
 	Namespace  = "openshift-azure-operator"
 	SecretName = "cluster"
+
+	OperatorIdentityName       = "ServiceOperator"
```

Comment:
Do we have a naming rule for the identity name?
`ServiceOperator` doesn't sound the identity for `aro-operator` for me.